aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncGraphQLRequestFactory.java

```diff
@@ -204,7 +204,11 @@ private AppSyncGraphQLRequestFactory() { }
             }
 
             if (!QueryPredicates.all().equals(predicate)) {
-                String conditionType = "Model" + graphQlTypeName + "ConditionInput";
+                String conditionType = new StringBuilder()
+                    .append("Model")
```

Comment:
Not the subclass simple name?

Reply:
graphQLTypeName is actually the same as the simpleName(), it's just built in a more roundabout way:

```
String graphQLTypeName = ModelSchema.fromModelClass(model.getClass()).getName()
```

In `ModelSchema.fromModelClass`, `name` is obtained via `modelClass.getSimpleName()`